lib/babe/babe.go

```diff
@@ -185,13 +179,16 @@ func (b *Service) setEpochData(cfg *ServiceConfig, genCfg *types.BabeConfigurati
 		}
 	}
 
-	if cfg.Threshold == nil {
+	if cfg.ThresholdNumerator == 0 && cfg.ThresholdDenominator == 0 {
```

Comment:
can you change this to be if `cfg.ThresholdDenominator == 0` only? since we should allow numerator to be 0